docs/Systems/Tailwind/GettingStarted.stories.mdx

```diff
@@ -71,7 +71,7 @@ These directives inject classes that are needed for certain Tailwind features. F
 
 ### Tailwind Play
 
-Tailwind Play is an online code sandbox with Tailwind set up out of the box. The Kaizen Team has created an instance with the Kaizen Preset values [here](https://play.tailwindcss.com/dy7PzIcLed)
+Tailwind Play is an online code sandbox with Tailwind set up out of the box. The Kaizen Team has created an instance with the Kaizen Preset values [here](https://play.tailwindcss.com/AAjA6HqO5Y)
```

Comment:
Sorry to be nitpicking here, but the example in the Playground is using text color with opacity 
<img width="361" alt="image" src="https://user-images.githubusercontent.com/763385/222039281-b374ad53-4ce9-4984-9ccc-a8251acb8e55.png">
which is something we wanted to discourage (and remove from our current `Typography` components).
Thus I think we shouldn't have it in our official examples.

Reply:
Thanks that's interesting... Let me see where that's coming from.